aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -284,6 +324,21 @@ void slow_conv2d_backward_out_cpu_template(
       false);
 
   const Tensor input = input_.contiguous();
+
+  // Compute shape of columnized data excluding batch dim.
+  const int64_t dim_planes = 1;
+  const int64_t dim_height = 2;
+  const int64_t dim_width = 3;
+  const int64_t n_input_plane = input.size(dim_planes);
+  const int64_t input_height = input.size(dim_height);
+  const int64_t input_width = input.size(dim_width);
+  const int64_t output_height =
+      (input_height + 2 * pad_height - kernel_height) / stride_height + 1;
+  const int64_t output_width =
+      (input_width + 2 * pad_width - kernel_width) / stride_width + 1;
+  const int64_t fgrad_input_size =
+      n_input_plane * kernel_height * kernel_width * output_height * output_width;
```

Comment:
nit: this shares code with compute_columns2d. We should try to deduplicate (maybe by adding this to a helper function somewhere?)

Reply:
I think the `output_width` / `output_height` calculations would make sense to be pulled out into a helper function